setup.py

```diff
@@ -54,14 +54,16 @@ def get_description():
     author_email='bemcdonnell@gmail.com',
     install_requires=REQUIREMENTS,
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
+    package_data={'': ['lib/windows/swmm5.dll', 'lib/macos/swmm5.so' ,
```

Comment:
@abhiramm7, it looks like `package_data` is being overwritten.  add it into the list